.gitignore

```diff
@@ -80,13 +80,13 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
```

Comment:
Yeah - Grunt is a buildtool, we use Gulp so Grunt will never become part of our dev env.